### PR TITLE
rotate bundle

### PR DESCRIPTION
**Pull request by denis-yuen**
**Description**
Rotate encrypted test bundle

**Review Instructions**
Circle ci builds should work

**Issue**
n/a

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that you pass the basic style checks and unit tests by running `mvn clean install`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] Follow the existing JPA patterns for queries, using named parameters, to avoid SQL injection
- [x] If you are changing dependencies, check the Snyk status check or the dashboard to ensure you are not introducing new high/critical vulnerabilities
- [x] Assume that inputs to the API can be malicious, and sanitize and/or check for Denial of Service type values, e.g., massive sizes
- [x] Do not serve user-uploaded binary images through the Dockstore API
- [x] Ensure that endpoints that only allow privileged access enforce that with the `@RolesAllowed` annotation
- [x] Do not create cookies, although this may change in the future
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

File: encrypt.sh [hunk 1]
```diff
@@ -7,12 +7,12 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
-: "$CIRCLE_CI_KEY_2"
-: "$CIRCLE_CI_IV_2"
+: "$CIRCLE_CI_WS_KEY"
```
Comment (kathy-t) [inline encrypt.sh:10]: Comment on line 3 needs to be updated with new environment variable names

File: scripts/decrypt.sh [hunk 2]
```diff
@@ -9,10 +9,10 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
-: "$CIRCLE_CI_KEY_2"
-: "$CIRCLE_CI_IV_2"
+: "$CIRCLE_CI_WS_KEY"
```
Comment (kathy-t) [inline scripts/decrypt.sh:12]: Comment on line 4 needs to be updated

**Review by kathy-t — APPROVED** [review]

**Review by coverbeck — APPROVED** [review]
The skip ci is the right thing to do, but it's probably going to break our nightly deploys when merged, alas. Unless we fixed that in the deployer.

**Comment by denis-yuen** [comment]
> The skip ci is the right thing to do, but it's probably going to break our nightly deploys when merged, alas. Unless we fixed that in the deployer.

Tried to edit the message on the squash commit to force a buiild

**Pull request merged**